### ISSUE TITLE
Simplified regression tutorial

### ISSUE THREAD

**Issue by radeusgd** [Luna-Tensorflow/Luna-Tensorflow#84]
We want to reproduce the regression tutorial as in: https://www.tensorflow.org/tutorials/keras/basic_regression

Due to the fact that data loading and cleanup is out of our scope (it's for Dataframes), in this ticket we can skip these phases and just load data that's been pre-cleaned in Python, so that the Luna example only needs to include:
- loading data
- training the network
- evaluating the results on test set

The result of this task should be a self-contained project that can be opened in Luna Studio.
The data should be packaged with it or possible to download somehow.

**Comment by radeusgd** [comment]
The tutorial is being developed in https://github.com/Luna-Tensorflow/RegressionTutorial

For now several things are missing and will be added in a separate issue after next milestone:
- feature normalization (it's more a Dataframes concern, but we could provide some helper functions, see: https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html)
- early stopping in training
